### ISSUE TITLE
Unknown version 4.0.1 of electron

### ISSUE THREAD

**Issue by comatory** [browserslist/browserslist#351]
I'm adding this to my `package.json`:

```
  "browserslist": [
    "node 10.15.0",
    "Chrome >= 69",
    "Firefox >= 63",
    "Safari >= 11",
    "electron 4.0.1"
  ]
```

That version of electron is not recognized. How do I make this work? I basically want to use babel preset env but have the config in `package.json`

**Comment by ai** [comment]
We are using this package for Electron versions https://github.com/kilian/electron-to-chromium

It has 4.0.1 version https://github.com/Kilian/electron-to-chromium/blob/master/full-versions.js#L14

You need to update your npm dependencies by `npm update`.

**Comment by Noitidart** [comment]
I had similar problem with using `5.0.2`. I ended up using `5.0` and it worked.

**Comment by ai** [comment]
@Noitidart did you try `npm update`?

**Comment by Noitidart** [comment]
Thanks @ai for the reply. Yep I tried it and I even listed it out to show the versions and they are very latest:

```
PS C:\Users\Mercurius\Documents\GitHub\rxjse> npm list electron-to-chromium
yanky-panky@1.0.0 C:\Users\Mercurius\Documents\GitHub\rxjse
`-- @babel/preset-env@7.4.5
  `-- browserslist@4.6.0
    `-- electron-to-chromium@1.3.137
```

**Comment by ai** [comment]
Hm. It is the latest. Can you open an issue at https://github.com/kilian/electron-to-chromium

**Comment by Noitidart** [comment]
Thanks! Done

**Comment by Kilian** [comment]
hi @ai! I think all of this is because you import versions.js and not full-versions. Versions doesn't include patch versions, as they never change chromium versions (nor do minor versions, but that's there for pre-1.0 when it did). So, browserlist only supports the X.X notation and not the X.X.X notation. Maybe you can either ignore the last `.X` in electron queries? Then you don't have to import the full-versions.

**Comment by ai** [comment]
@kilian can you send PR?

**Comment by Noitidart** [comment]
Thanks very much @Kilian for trying to help out!!

**Comment by Kilian** [comment]
@ai yeah: https://github.com/browserslist/browserslist/pull/378 also fixes https://github.com/browserslist/browserslist/issues/326#issuecomment-445048906

**Comment by ai** [comment]
Released in 4.6.1
